viewer/multies.js

```diff
@@ -117,7 +117,14 @@ function node2Prefix(node) {
   return "";
 }
 
+var activeESNodes = [];
+
+function getActiveNodes(){
+  return activeESNodes.slice();
```

Comment:
why slice?

Reply:
**activeESNodes.slice()** returns the elements (values) of an array instead of the reference to the array. The statement **activeESNodes.slice()** will make the list activeESNodes Thread-safe. If the statment **return activeESNodes** is used then it is possible that the list activeESNodes is getting updated by the **enumerateActiveNodes** method while the list is being used by the **SimpleGather** or **Msearch** method.